### DISCUSSION TITLE
How to upgrade the existing MiroMDM with new release?

### DISCUSSION THREAD

**Discussion by honeyankit** [micromdm/micromdm#804] [Q&A] [answered]
Q. I do not see any documentation related to upgrading the path of MicoMDM to the new release?

**Comment by jessepeterson** [comment, marked as answer]
Hello! You're right, there isn't much documentation around upgrading, unfortunately. However the upgrade path is very straight forward: just replace the old binaries with the new ones and restart the server. There haven't been any breaking changes or migrations that need to be run for the life of the project and if there ever were, I'm sure we'd call them out on the releases page. Cheers!

**Reply by honeyankit** [reply]
Thank you for the answer. So nothing changes with respect to the MDM server config (~/.micromdm/servers.json). Also, we do not need to worry about any of the apple push certification processes after the upgrade?

**Reply by jessepeterson** [reply]
No, none of that should change. As long as your push certificate is still valid you should be fine. To be clear I would definitely _backup_ your `micromdm.db` before the upgrade just in case, but in general it's an upgrade-in-place sort of thing.

**Comment by honeyankit** [comment]
Thank you for the answer. So nothing changes with respect to the MDM server config (~/.micromdm/servers.json). Also, we do not need to worry about any of the apple push certification processes after the upgrade?
